### PR TITLE
[RBFDD-449] Document Flink SBT

### PR DESCRIPTION
**Pull request by stefanobaghino**
Remove build.properties, add assembly.sbt

Use code from the rbp-docs-code repo

### CODE REVIEW

**Comment by stefanobaghino** [comment]
Superseded by #14.

**Comment by stefanobaghino** [comment]
@giampaolotrapasso Open for review. Will use #14 for Kafka only moving forward.

File: flink/flink.rst [hunk 1]
```diff
@@ -6,12 +6,10 @@ Flink
 
 The following documentation will target RBP-specific features: where not else stated, please refer to the `official documentation <https://ci.apache.org/projects/flink/flink-docs-release-1.0/>`_.
```
Comment (giampaolotrapasso) [inline flink/flink.rst]: RBP should be RBFDD

File: flink/sbt/sbt.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
+===============================
+Using SBT to create a Flink job
+===============================
+
+Motivation
+----------
+
+Maven is a widely accepted build tool for projects that target the Java platform. The Radicalbit Fast Data Distribution provides an archetype to kickstart your development projects using Maven, but Scala developers may find more compelling to use the Maven alternative of choice for the Scala ecosystem, SBT (Simple Build Tool).
```
Comment (giampaolotrapasso) [inline flink/sbt/sbt.rst]: Maven/SBT links would improve doc. Spitting into two sentences may improve readability.

File: flink/sbt/sbt.rst [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+Basic 'archetype'
+-----------------
+
+Currently, SBT does not provide built-in capabilities to scaffold a project like Maven archetypes do.
```
Comment (giampaolotrapasso) [inline flink/sbt/sbt.rst:13]: With the possibility of creating more examples in Scala, maybe we want to evaluate [this plugin](https://github.com/softprops/np#for-sbt-013) that allows with a single line in sbt configuration, to create an empty Scala project.

File: flink/sbt/sbt.rst [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+As such, some manual work is required to initialize the build definition properly.
```
Comment (giampaolotrapasso) [inline flink/sbt/sbt.rst:14]: Why not giving user the url of the code? Probably is more simple to checkout that repo and run the provided example. Explanations for manual creation of the project as already detailed below could follow.

File: flink/sbt/sbt.rst [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    mkdir my-flink-project
+
+Then, move into the directory and create a project directory::
+
+    cd my-flink-project
+    mkdir project
+
+To package our application, we'll use the ``sbt-assembly`` plugin. To add it to our build, create and open in your favorite text editor the ``project/assembly.sbt`` file and fill it in with this single line:
+
+.. literalinclude:: /rbp-docs-code/sbt/project/assembly.sbt
```
Comment (giampaolotrapasso) [inline flink/sbt/sbt.rst]: something is wrong with modules, cannot find referenced code for this link and following.

File: flink/sbt/sbt.rst (hunk 3 above)
Comment (stefanobaghino) [inline flink/sbt/sbt.rst:13]: I agree; furthermore, SBT 1.0 should introduce native scaffolding. For the time being, I would keep things simple and not require users to adopt plugins.

File: flink/sbt/sbt.rst (hunk 4 above)
Comment (stefanobaghino) [inline flink/sbt/sbt.rst:14]: I agree, but can we postpone this untile the code repository name stabilizes?

Comment (giampaolotrapasso) [inline flink/sbt/sbt.rst:14]: Please, put anyway the link. I prefer changing the link that forgetting it. I will take care of name refactoring.

Comment (stefanobaghino) [inline flink/sbt/sbt.rst:14]: Ok, I'll add a link to the source file since the project is shared among several examples. I'll open an issue to update it.

**Comment by stefanobaghino** [comment]
This should address all remarks.

**Comment by giampaolotrapasso** [comment]
Thanks!

**Pull request merged**